src/components/Navigation/TopBar.tsx

```diff
@@ -43,7 +43,7 @@ function TopBar({breadcrumbLabel, activeWorkspaceID, shouldDisplaySearch = true,
     const displaySearch = !isAnonymousUser && shouldDisplaySearch;
 
     return (
-        <View style={styles.w100}>
+        <View style={styles.topBarWrapper}>
```

Comment:
I think what I don't understand is... why do we need some kind of wrapper here with margin? Why can't we just use absolute position and have the loading bar just appear directly below the header portion of the LHN? If there is a 12px gap between the header and the first LHN item, we have plenty of space (12px worth) to place a 2px tall loading bar.

Reply:
Okay great, where are we at then? Should we run a test build to see if these changes will work everywhere?

Reply:
Hmm I don't think we want to add a border if one doesn't exist already. Can you show me in screenshots what the bug is?

Reply:
Hmm but wouldn't we be able to fix that with absolute position and making sure it has a higher z-index so it doesn't get cut off by the list behind it?

Reply:
I'm not sure what to say then - we need to fix the bug somehow. Do you have any other ideas?